src/backupNg/backupNg.spec.js

```diff
@@ -169,5 +169,64 @@ describe("backupNg", () => {
       });
       expect(log.warnings).toMatchSnapshot();
     });
+
+    it("fails trying to run a backup job with a VM without disks", async () => {
+      const { id: template } = find(xo.objects.all, { type: "VM-template" });
+      const vmIdWithoutDisks = await xo.createTempVm({
+        name_label: "XO Test Without Disks",
+        name_description: "Creating a vm without disks",
+        template,
+      });
+
+      const scheduleTempId = randomId();
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleTempId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
```

Comment:
Please set report when to always.

Reply:
I'm sorry. I withdraw what i said. 
We should set report when to never because we aren't able to validate the report automatically and to not interfere with the backup report.